<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #000;</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>173</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Server">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Server">
      <item>
       <widget class="QLabel" name="label_Port">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_Port">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="minimum">
         <number>1111</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>8888</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Start">
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;
}
QPushButton::hover{
	background: #0f0;
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_State">
      <property name="styleSheet">
       <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 12px; border: 2px inset #ff0;</string>
      </property>
      <property name="text">
       <string>State</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_Party">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Party">
        <item>
         <widget class="QLabel" name="label_Player">
          <property name="styleSheet">
           <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
          </property>
          <property name="text">
           <string>Player(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_Player">
          <property name="styleSheet">
           <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Party">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Quit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;
}
QPushButton::hover{
	background: #f00;
}</string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quit</sender>
   <signal>clicked()</signal>
   <receiver>Log</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
